elements/lisk-transactions/src/utils/validation/validation.ts

```diff
@@ -160,6 +161,9 @@ export const validateTransferAmount = (data: string) =>
 	isGreaterThanZero(new BigNum(data)) &&
 	!isGreaterThanMaxTransactionAmount(new BigNum(data));
 
+export const validateTransferData = (data: string): boolean =>
```

Comment:
I think, in general `validate` keyword is used to throw something right now.
Maybe change to `isvalidTransferData`?